content/user/email/add_message.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Adding a message to the checkout email
+description: How do I add a message to my order confirmation emails? 
+category: email
+weight: 10
+---
+
+In Zen Cart 1.5.6, a plugin called [Order Message](https://www.zen-cart.com/downloads.php?do=file&id=2200) was pulled into the Zen Cart core.  To use it, 
```

Comment:
Can we reword this article to "present day tone"? ie: "to add a static message to every email, define (the constant, in the file); If you're using old than v1.5.6, install the plugin to get the same functionality...", or something to that effect